client/src/modules/journal/templates/grid.footer.html

```diff
@@ -1,4 +1,6 @@
-<div style="padding: 0.5rem; font-weight: bold;">
-  <span translate>FORM.INFO.SHOWING</span>
-  {{ grid.appScope.numberCurrentGridTransactions }} <span translate>FORM.INFO.OF</span> {{ grid.appScope.numberTotalSystemTransactions }} <span translate>FORM.INFO.UNPOSTED_TRANSACTION</span>
+<div class="ui-grid-cell-contents">
+  <span>({{grid.rows.length}} <span translate>FORM.INFO.ROWS</span>)</span>
+  <span style="color : #FDBA64" class="fa fa-circle"></span> {{grid.appScope.rowsDetails.totalPosted}} <span translate>FORM.LABELS.POSTED_TRANSACTIONS</span>
```

Comment:
Can this color be a class?  That way we can change it in a single place if we never need to change it in the future ... such as `bhima-bootstrap.less`.

Also, since it's shared in multiple places (and presumably will be shared in the Account Statement and others), putting in a class early will make sure we have consistency.